Makefile

```diff
@@ -59,9 +61,16 @@ formatter:
 	poetry run black rasa tests
 
 lint:
-	poetry run flake8 rasa tests
+    # Ignore docstring errors when running on the entire project
+	poetry run flake8 rasa tests --extend-ignore D
 	poetry run black --check rasa tests
 
+branch ?= master
+lint-docs:
+    # Lint docstrings only against the the diff to avoid too many errors.
+    # Check only production code. Ignore other flake errors which are captured by `lint`
```

Comment:
```suggestion
     # Lint docstrings only against the the diff to avoid too many errors.
     # Check only production code. Ignore other flake errors which are captured by `lint`
```